### DISCUSSION TITLE
how to use sendgrid as email provider

### DISCUSSION THREAD

**Discussion by ahmedabdelhafez** [supabase/supabase#10966] [General]
Hello dear,
i have long time search for a way to use sendgrid as smtp email provider, and to send email from it also using supabase
can any one help with tested solution

thanks.

**Comment by Hallidayo** [comment]
Hi @ahmedabdelhafez - I've had a look and I can not see any docs on Supabase and SendGrid.

I've also had a look and seen you've created issue #10030 for this.

I can look into this and have a look at creating some documentation.

**Reply by ahmedabdelhafez** [reply]
This tutorial in the previous issue that I was created not working with me and now supabase added the SMTP and I need any tutorial to use it or implement it with any provider such as send grid , mailgun , Aws ses

**Comment by Hallidayo** [comment]
Is this the [supabase-mailer](https://github.com/burggraf/supabase-mailer) tutorial that you've got an issue with?
